File: sdkwork-app-sdk-csharp/Models/RealtimeAuditForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class RealtimeAuditForm
    {
        public string? StreamId { get; set; }
        public string? StreamUrl { get; set; }
        public string? StreamType { get; set; }
        public string? RoomId { get; set; }
        public string? PublisherId { get; set; }
        public List<string>? CheckTypes { get; set; }
        public string? CallbackUrl { get; set; }
        public bool? CheckAudio { get; set; }
        public bool? CheckVideo { get; set; }
    }
}

File: sdkwork-app-sdk-go/types/realtime_audit_form.go
package types

// 实时内容审核请求
type RealtimeAuditForm struct {
	StreamId string `json:"streamId"`
	StreamUrl string `json:"streamUrl"`
	StreamType string `json:"streamType"`
	RoomId string `json:"roomId"`
	PublisherId string `json:"publisherId"`
	CheckTypes []string `json:"checkTypes"`
	CallbackUrl string `json:"callbackUrl"`
	CheckAudio bool `json:"checkAudio"`
	CheckVideo bool `json:"checkVideo"`
}
